portfolio/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Toya </title>
```

Comment:
```suggestion
<title>Toya</title>
```

Hey Toya, there's an extra space at the end of your name - you can delete it if you like. Otherwise, lgtm!